operator/docs/forwarding_logs_without_gateway.md

```diff
@@ -0,0 +1,124 @@
+
+__Please read the [hacking guide](./hack_loki_operator.md) before proceeding with the following instructions.__
+
+_Note:_ This document is meant to be used with OpenShift only.
+
+_Disclaimer:_ This document helps in connecting fluentd or vector to LokiStack without authentication/authorization.
+
+## Openshift Logging
+
+[Openshift Logging](https://github.com/openshift/cluster-logging-operator) supports [forwarding logs to an external Loki instance](https://docs.openshift.com/container-platform/4.9/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-loki_cluster-logging-external) using fluentd and vector as log forwarders.
+The below step-by-step guide will help you to send application, infrastructure, and audit logs to the LokiStack through the Distributor endpoint.
+The steps remain same for both fluentd and vector.
+
+In order to enable communication between the client(s) and the gateway, follow these steps:
+
+* Deploy the Loki Operator and a `lokistack` instance for [OpenShift](./hack_loki_operator.md#hacking-on-loki-operator-on-openshift).
+
+* Deploy the OpenShift Logging Operator from the Operator Hub or using the following command locally:
+
+    ```console
+    make deploy-image deploy-catalog install
+    ```
+  
+* Create a Cluster Logging instance in the `openshift-logging` namespace with only `collection` defined.
+
+    For fluentd:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "fluentd"
+          fluentd: {}
+    ```
+
+    For vector:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "vector"
+          fluentd: {}
+    ```
+
+* By default, the TLS communication is on for communication with the distributor. Therefore, you need to create a secret. The secret must exist in the openshift-logging namespace, and must have a key `ca-bundle.crt`.
+
+    This key already exists when LokiStack instance got created but as a Config Map. Hence, you need to fetch them and put into a secret file.
+
+    Fetch the `ca-bundle.crt` using:
+
+    ```console
+    kubectl -n openshift-logging get cm lokistack-gateway-lokistack-dev-ca-bundle -o jsonpath="{.data.service-ca\.crt}" | base64 -w 0
+    ```
+  
+* Once secret is fetched, create a new secret file:
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: loki-distributor-metrics
+      namespace: openshift-logging
+    type: Opaque
+    data:
+      ca-bundle.crt: <CA_BUNDLE>
+    ```
+    
+    where `<CA_BUNDLE>` is the copied value from previous step.
+
+* Create the above secret file in the cluster:
+
+    ```console
+    kubectl create -f <SECRET_FILE_NAME>
```

Comment:
Is it maybe simpler to write the CA to a file and use `oc create secret generic $name --from-file` instead? That way kubectl is also doing the base64-encoding.